README.md

```diff
@@ -38,15 +38,15 @@ Gardener takes part in the [Certified Kubernetes Conformance Program](https://ww
 
 Continuous conformance test results of the latest stable Gardener release are uploaded regularly to the CNCF test grid:
 
-| Provider/K8s | v1.25 | v1.24 | v1.23 | v1.22 | v1.21 | v1.20 | v1.19 | v1.18 | v1.17 |
```

Comment:
You could update the version in https://github.com/gardener/gardener/blob/30ce9c0f4d3fcbbfb496236b868eccc09a97defa/docs/development/local_setup.md#L468 to a more recent one.

For example 
```
kubectl get seed
NAME       STATUS    PROVIDER    REGION      AGE    VERSION       K8S VERSION
seed-aws   Ready     aws         eu-west-1   4m     v1.61.0-dev   v1.24.8
```